diracx-logic/src/diracx/logic/jobs/query.py

```diff
@@ -40,7 +40,10 @@ async def search(
     if query_logging_info := ("LoggingInfo" in (body.parameters or [])):
         if body.parameters:
             body.parameters.remove("LoggingInfo")
-        body.parameters = ["JobID"] + (body.parameters or [])
+            if not body.parameters:
+                body.parameters = None
```

Comment:
Looks like an impossible thing to happen

Reply:
This in essence forces `body.parameters` to `None` if removing `LoggingInfo` leaves an empty list, hence making it as if the default set of parameters (in addition to the logging info) are the ones should be queried when calling `job_db.search`. I ran into an unfortunate edge case in getting the unit tests to pass and this is what I came up with.

Reply:
The default parameter set includes `JobID` and a bunch of other things, but if you don't keep this then you end up with `body.parameters = ["JobID"]` only